src/cachingreader.cpp

```diff
@@ -162,66 +159,90 @@ void CachingReader::freeAllChunks() {
     m_allocatedChunks.clear();
     m_mruChunk = NULL;
 
-    QSet<Chunk*> reserved = QSet<Chunk*>::fromList(m_chunksBeingRead.values());
+    for (QVector<Chunk*>::const_iterator it = m_chunks.constBegin();
+         it != m_chunks.constEnd(); ++it) {
```

Comment:
> Lets say at least 8 and everyone should be happy.

I wouldn't be happy with that. I like how hanging indents line up at the open parens -- this is very logical.

```c++
functionCall(foo,
             bar,
             baz);

if (asdf && qwer &&
    zxcv) {

}

for (int longVarName = 0; 
     longVarName < reallyLongLongLongConstant;
     ++longVarName) {

}
```